src/parse.js

```
@@ -141,8 +141,15 @@ function parse(source, root, options) {
         do {
             if (acceptStrings && ((token = peek()) === "\"" || token === "'"))
                 target.push(readString());
-            else
-                target.push([ start = parseId(next()), skip("to", true) ? parseId(next()) : start ]);
+            else {
+                try {
+                    target.push([ start = parseId(next()), skip("to", true) ? parseId(next()) : start ]);
+                } catch (err) {
+                    if (typeRefRe.test(token) && edition) {
+                        target.push(token);
+                    }
```

Should we rethrow the error otherwise?